docs/release-notes/february-2024.md

```diff
@@ -0,0 +1,172 @@
+
+* The following versions of the OSS Engine were released in February 2024:
+  * [<i class="fas fa-external-link fa-xs"></i>1.59.1](https://github.com/semgrep/semgrep/releases/tag/v1.59.1)
+  * [<i class="fas fa-external-link fa-xs"></i>1.60.0](https://github.com/semgrep/semgrep/releases/tag/v1.60.0)
+  * [<i class="fas fa-external-link fa-xs"></i>1.60.1](https://github.com/semgrep/semgrep/releases/tag/v1.60.1)
+  * [<i class="fas fa-external-link fa-xs"></i>1.61.1](https://github.com/semgrep/semgrep/releases/tag/v1.61.1)
+  * [<i class="fas fa-external-link fa-xs"></i>1.62.0](https://github.com/semgrep/semgrep/releases/tag/v1.62.0)
+  * [<i class="fas fa-external-link fa-xs"></i>1.63.0](https://github.com/semgrep/semgrep/releases/tag/v1.63.0)
+
+## 🌐 Cloud Platform
+
+### Added
+
+- API: Added a `rule` object under `findings` with the following fields: 
+    - `name`
+    - `message`
+    - `confidence`
+    - `category`
+    - `subcategories`
+    - `technologies`
+    - `vulnerability_classes`
+    - `cwe_names`
+    - `owasp_names` <!-- 12868 --> 
+- Added distinction between Pro engine and OSS findings in the Playground and Editor. <!-- 12275 -->
+- Support `linux-arm64` platform for Semgrep Pro Engine downloads <!-- 12430 -->
+
+### Changed
+
+- Updated the Semgrep Cloud Platform login page. <!-- 12744 -->
+- Updated the login process from the CLI to SCP. This change affects new users. <!-- 12531 -->
+- Updated the Semgrep installation instructions for Docker. <!-- 12531 -->
+- Improved performance of Semgrep Playground and Editor. <!-- 12461 -->
+
+### Fixed
+
+- Fixed a bug where the navigation sidebar covered mobile screens and could not be collapsed. <!-- 12876 -->
+
+## 💻 Code
+
+### Added
+
+<!-- I think the highest item should probably be our C, C++ launch? -->
+* Added new rules for Elixir and the Phoenix framework, covering various security and correctness issues. These are available in the `p/elixir`
+  ruleset. <!-- Do we need to mention that this is for Pro users only? -->
+* Added support for Python, with a focus on the Flask ecosystem, to the Semgrep
+  Pro Engine.
+* Added ability to collapse and turn off patterns in a rule.
+* Added the ability to distinguish between which rules are available to Semgrep
+  Pro Engine users and which ones are available to Semgrep OSS users.
+* Added support for nested record patterns on the left-hand side of an
+  assignment.
+* `metavariable-regex` can now match on metavariables of interpolated strings
+  that use variables with known values.
+* Added support for parsing Swift Package Manager manifest files and lock files.
+* **Taint analysis**:
+  * Added support for Python constructors
+  * Added support for index sensitivity. Semgrep tracks taint on individual
+    indexes of a data structure when these are constant values, either integers
+    or strings, and the code uses the built-in syntax for array indexing.
+  * Added `exact: false` so you can specify that anything inside a code
```

Comment:
```suggestion
  * Added `exact: false` so you can specify that anything inside a code
```
I believe this is nested under one of the [required fields](https://semgrep.dev/docs/writing-rules/rule-syntax/#required)? Do you know if there's an example rule using `exact: false` so we can check?